src/core/lib/security/security_connector/ssl_utils.cc

```diff
@@ -67,6 +67,9 @@ static const char* cipher_suites = nullptr;
 // All cipher suites for default are compliant with HTTP2.
 GPR_GLOBAL_CONFIG_DEFINE_STRING(
     grpc_ssl_cipher_suites,
+    "TLS_AES_128_GCM_SHA256:"
+    "TLS_AES_256_GCM_SHA384:"
+    "TLS_CHACHA20_POLY1305_SHA256:"
```

Comment:
Poking around the code, it looks like this also is just used in `SSL_CTX_set_cipher_list`? In that case, while adding them isn't harmful, I don't think it does anything. (For the same reason as the other cipher lists.)

Reply:
Thanks for catching this - I've removed them in the newest commit.

Reply:
It turns out these were also used for some sanity checking (when I pushed the most recent commit, one of the Kokoro tests started failing) so I added them back in.

Reply:
Sounds good. (I guess some check that the resulting cipher was one of the configured ones?)